tensorboard/webapp/widgets/data_table/data_table_component.scss

```diff
@@ -55,7 +67,40 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
   justify-content: center;
 }
 
-.add-button-cell {
-  display: table-cell;
-  width: 40px;
+.left-section {
+  flex-grow: 1;
+}
+
+.right-section {
+  background-color: mat.get-color-from-palette($tb-background, background);
+  position: sticky;
+  right: -1px; // Prevent fractional width from creating a gap.
+  z-index: 1;
+
+  @include tb-dark-theme {
+    background-color: map-get($tb-dark-background, background);
+  }
+
+  .add-button-column {
+    width: 40px;
+    height: 100%;
+
+    .add-button {
+      position: sticky;
+      top: 0;
+    }
+  }
+}
+
+.right-section .add-button-column.small-add-button {
+  display: flex;
+  justify-content: center;
+  width: 24px;
+
+  .add-button {
+    // Override very high specificity with !important
+    // Ref: https://github.com/tensorflow/tensorboard/blob/f2e8bd82a533c175f6dc5be3da65344d719ae0f6/tensorboard/webapp/theme/_tb_theme.template.scss#L320
+    width: 20px !important;
```

Comment:
I'm pretty opposed to using `!important` as it eventually spirals into an arms race.
If you updated the template to use a css variable, then changed the value of the variable here you wouldn't have to use `!important`.

Reply:
Oh this is great, done!